plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/properties/ObjectPropertyDescriptor.java

```diff
@@ -243,11 +252,19 @@ public String[] getFeatures() {
         if (this.isHref()) features.add(DBConstants.PROP_FEATURE_HREF);
         if (this.isViewable()) features.add(DBConstants.PROP_FEATURE_VIEWABLE);
         if (this.isPassword()) features.add(DBConstants.PROP_FEATURE_PASSWORD);
+
         return features.toArray(new String[0]);
     }
 
     @Override
     public boolean hasFeature(@NotNull String feature) {
+
+        List<String> features = parseFeatures();
```

Comment:
Function hasFeature may be called 1000 times per second. It is not good to parse, create array, etc for every call.